app/models/minute.rb

```diff
@@ -3,6 +3,9 @@
 class Minute < ApplicationRecord
   CLONED_BOOTCAMP_WIKI_PATH = Rails.root.join('bootcamp_wiki_repository').freeze
   CLONED_AGENT_WIKI_PATH = Rails.root.join('agent_wiki_repository').freeze
+  DEFAULT_BRANCH_FOR_GITHUB_WIKI = 'master'
+  CLOCK_DRIFT_BUFFER = 60
+  TOKEN_EXPIRATION_TIME = 600
```

Comment:
```suggestion
  CLOCK_DRIFT_BUFFER_SEC = 60
  TOKEN_EXPIRATION_TIME_SEC = 600
```

時間の数値って、ミリ秒か秒か分か時間か日か、単位がパッとわからないときが多いので、定数名に単位も含めておくのがお勧めです。

Reply:
単位があった方がわかりやすいですね！
[5780a60](https://github.com/Kassy0220/fjord-minutes/pull/368/commits/5780a60b8e12ada12504834b50952ae7075d592a) で定数名に単位を含めるようにしました。